documents/motions/16.2021.1 - Disputes.md

```diff
@@ -1,20 +1,19 @@
 # SUBMISSION OF PROPOSED MOTION
 
-**Motion number:** 16.2021.1  
+**Motion number:** 16.2024.1  
 **Subject:** Disputes  
 **Intent:** Definition of the Processes in case of Disputes  
 **Submitted by:** WCA Board  
-**Date:** February 28, 2021  
+**Date:** TBC, 2024  
 
 # Motion
 
-1. All acts and decisions of WCA Staff may be subject to disputes by affected individuals.
-   1. Disputes regarding the WCA Regulations must be directed to and handled by WCA Regulations Committee, and are bound by the provisions of the WCA Regulations.
-      1. Appeals regarding the final decision must be directed to and handled by the WCA Board.
-   2. Disputes regarding the WCA Bylaws and the Motions should be directed to and handled by the WCA Board, and are bound by the provisions of the WCA Bylaws and the Motions and the applicable governing laws.
-      1. Appeals regarding the final decision should be directed to and handled by WCA Ethics Committee.
-   3. All other Disputes should be directed to and handled by WCA Disciplinary Committee, and are bound by the WCA Regulations, the WCA Bylaws and the Motions, and the applicable governing laws.
-      1. Appeals regarding the final decision should be directed to and handled by WCA Ethics Committee.
+1. Acts and decisions of WCA Staff may be subject to disputes by affected individuals or organizations.
+   1. Disputes regarding the application of WCA Regulations must be directed to and handled by WCA Regulations Committee, and are bound by the provisions of the WCA Regulations.
+   2. Disputes regarding the conduct of WCA Staff where it is alleged the Spirit, Mission, Motions, Policies, Codes, Regulations and the applicable governing laws are violated must be directed to and handled by WCA Integrity Committee, and are bound by the Spirit, Mission, Motions, Policies, Codes, Regulations of the WCA, and the applicable governing laws.
+   3. Disputes regarding the conduct of WCA Community members where it is alleged the Spirit, Mission, Motions, Policies, Codes, Regulations and the applicable governing laws are violated must be directed to and handled by WCA Integrity Committee, and are bound by the Spirit, Mission, Motions, Policies, Codes, Regulations of the WCA, and the applicable governing laws. 
+   4. All other Disputes should be directed to and handled by WCA Integrity Committee, and are bound by the Spirit, Mission, Motions, Policies, Codes, Regulations of the WCA, and the applicable governing laws.
+2. All disputes may be subject to appeal by affected individuals or organizations.
```

Comment:
Instead of saying "all disputes may be subject to appeal" wouldn't it be better to say "The decision made by the Committee handling the dispute may be subject to appeal ... " - you're not appealing the dispute, you're appealing the decision made about it